_includes/tutorials/time-concepts/ksql/markup/prod/submit-to-api.adoc

```diff
@@ -0,0 +1,5 @@
+Launch your statements into production by sending them to the ksqlDB server REST endpoint with the following command:
```

Comment:
We don't use this either. Can we remove it?

Reply:
Removed